end-to-end-untargeted-metabolomics.Rmd

```diff
@@ -383,70 +398,94 @@ eic_Met <- chromatogram(data, rt = c(170, 200),
                            mz = Met_mz + c(-0.05, 0.05), aggregationFun = "max")
 ```
 
-Now we can `plot()` these EICs: 
+Now we can `plot()` these EICs:
 
 ```{r plot EIC, echo=FALSE}
 
 #' plot both EIC
 par(mfrow = c(1, 2))
-plot(eic_cystine, main = "Cystine", col = paste0(col_phenotype[bpc$phenotype], 80))
+plot(eic_cystine, main = "Cystine", col = paste0(col_sample, 80))
```

Comment:
easier/simpler to use this variable, because it provides already the colors in the order of the samples.